File: coletor/Registro.py
from SensorFluxo import SensorFluxo

class Registro:

    def __init__(self):
        self.id = ""
        self.valor = 0
        self.data = ""
        self.sensor = SensorFluxo()

    def setId(self, id):
        self.id = id

    def getId(self):
        return self.id

    def setValor(self, valor):
        self.valor = valor

    def getValor(self):
        return self.valor

    def setData(self, data):
        self.data = data

    def getData(self):
        return self.data

    def registrarSensor(self):
        self.valor = self.sensor.registrar()

File: servidor/src/java/modelo/Registro.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package modelo;

import java.util.Date;

/**
 *
 * @author osmar
 */
public class Registro extends Objeto{
    private double valor;
    private Date data;

    public Registro() {
    }

    public Registro(int id, double valor, Date data) {
        this.id = id;
        this.valor = valor;
        this.data = data;
    }

    @Override
    public int getId() {
        return id;
    }

    @Override
    public void setId(int id) {
        this.id = id;
    }

    public double getValor() {
        return valor;
    }

    public void setValor(double valor) {
        this.valor = valor;
    }
    
    public Date getData() {
        return data;
    }

    public void setData(Date data) {
        this.data = data;
    }
}
